__tests__/app.test.js

```diff
@@ -310,3 +310,28 @@ describe("DELETE /api/comments/:comment_id", () => {
   });
 });
 
+describe("GET /api/users", () => {
+  test("200: Responds with an array of user objects, each user object should have the following properties: username, name, avatar_url", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
```

Comment:
This is a good start, but dont forget to include an assertion before the forEach to ensure the array contains some elements. To ensure we dont get a false passing test if the array is empty. 

Test description doesnt need to list the properties since this can be found in your test body itself and is just repeated words